bcftools.wdl

```diff
@@ -55,3 +55,102 @@ task Bcf2Vcf {
+        memory: memory
+        docker: dockerImage
+    }
+
+    parameter_meta {
+        inputVcf: {description: "The vcf to be analysed.", category: "required"}
+        compareVcf: {description: "When inputVcf and compareVCF are given, the program generates separate stats for intersection and the complements. By default only sites are compared, samples must be given to include also sample columns.", category: "common"}
+        afBins: {description: "Allele frequency bins, a list (0.1,0.5,1) or a file (0.1\n0.5\n1).", category: "advanced"}
+        afTag: {description: "Allele frequency tag to use, by default estimated from AN,AC or GT.", category: "advanded"}
+        firstAlleleOnly: {description: "Include only 1st allele at multiallelic sites.", category: "advanced"}
+        collapse: {description: "Treat as identical records with <snps|indels|both|all|some|none>, see man page for details.", category: "advanced"}
+        depth: {description: "Depth distribution: min,max,bin size [0,500,1].", category: "advanced"}
+        exclude: {description: "Exclude sites for which the expression is true (see man page for details).", category: "advanced"}
+        exons: {description: "Tab-delimited file with exons for indel frameshifts (chr,from,to; 1-based, inclusive, bgzip compressed).", category: "advanced"}
+        applyFilters: {description: "Require at least one of the listed FILTER strings (e.g. \"PASS,.\").", category: "advanced"}
+        fastaRef: {description: "Faidx indexed reference sequence file to determine INDEL context.", category: "advanced"}
+        fastaRefIndex: {description: "Index file (.fai) for fastaRef. Must be supplied if fastaRef is supplied.", category: "advanced"}
+        include: {description: "Select sites for which the expression is true (see man page for details).", category: "advanced"}
+        splitByID: {description: "Collect stats for sites with ID separately (known vs novel).", category: "advanced"}
+        regions: {description: "Restrict to comma-separated list of regions.", category: "advanced"}
+        regionsFile: {description: "Restrict to regions listed in a file.", category: "advanced"}
+        samples: {description: "List of samples for sample stats, \"-\" to include all samples.", category: "advanced"}
+        samplesFile: {description: "File of samples to include.", category: "advanced"}
+        targets: {description: "Similar to regions but streams rather than index-jumps.", category: "advanced"}
+        targetsFile: {description: "Similar to regionsFile but streams rather than index-jumps.", category: "advanced"}
+        userTsTv: {description: "<TAG[:min:max:n]>. Collect Ts/Tv stats for any tag using the given binning [0:1:100].", category: "advanced"}
+        threads: {description: "Number of extra decompression threads [0].", category: "advanced"}
+        verbose: {description: "Produce verbose per-site and per-sample output.", category: "advanced"}
```

Comment:
There seems to be some inputs missing here, like dockerImage, the vcf indexes and outputDir.